### PR TITLE
Diagnostic log option

### PR DESCRIPTION
**Pull request by wanlwanl**

### CODE REVIEW

File: samples/ChatSample/ChatSample/appsettings.json [hunk 1]
```diff
@@ -2,7 +2,17 @@
   "Azure": {
     "SignalR": {
       "Enabled": true,
-      "ConnectionString": ""
+      "ConnectionString": "",
+      "DiagnosticLogs": [
```
Comment (vicancy) [inline samples/ChatSample/ChatSample/appsettings.json:6]: I am thinking if there is any way to leverage the `Logging` settings https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1

Comment (wanlwanl) [reply samples/ChatSample/ChatSample/appsettings.json:6]: But this option is about tracking client, and it's only an setting for Azure SignalR. Looks like it's fine to place in `Azure:SignalR` section. Maybe the improve the naming like this:
```
  "Azure": {
    "SignalR": {
      "Enabled": true,
      "ConnectionString": "",
      "TrackingClient": [
        {
          "LogType": "Messaging",
          "Accept": true
        },
        {
          "LogType": "Connectivity",
          "Accept": false
        }
      ]
    }
  }
```

**Comment by wanlwanl** [comment]
Consider only need to change the settings in server, no need to change anything in client.
Server controls what client matches the tracking rules.

```
  "Azure": {
    "SignalR": {
      ...
      "TrackingRules": [
        IP: ["x.x.x.x", "y.y.y.y"]
      ]
    }
  }
```

**Comment by wanlwanl** [comment]
Need to change client code and app server options.
Consider a website. Need to restart.

client query + server option:
client: `var conn = new HubConnectionBuilder().WithUrl($"{endpoint}?track=true")`
server: appsettings.json: `"acceptTracking":"true"`

**Comment by wanlwanl** [comment]
Add claim claims in client. No need to change server.
Consider a website. Need to restart.

**Comment by wanlwanl** [comment]
Finally, put all settings in server side, add a method like `ServerSticky`

**Pull request closed without merging**